SEM 2 PRACTICALS/prac_05/word_occurrences.py

```diff
@@ -0,0 +1,25 @@
+"""
+CP1404/CP5632 Practical
+word occurrences
+"""
+word_counter = {}
+word_list = []
+maximum_word_length = 0
+
+text = input("Text: ").lower()
+split_text = text.split(" ")
+for word in split_text:
+    word_list.append(word)
+    if word in word_counter:
+        word_counter[word] += 1
+    else:
+        word_counter[word] = 1
+sorted_list = sorted(word_list)
```

Comment:
Why keep two lists? You don't use the unsorted list. The new list has no meaningful context in the name ("words")